### ISSUE TITLE
Error in logs relating to temperature_set variable (wrong type)

### ISSUE THREAD

**Issue by malammar** [Hyundai-Kia-Connect/kia_uvo#956]
**Region and Brand of car**
America Hyundai (2023 Ioniq 5)

**Version of the integration**
2.24.0

**Describe the bug**
Checking logs I see an error related to the plugin.

**Debug logs if an error occurred**

```
home-assistant  | 2024-10-30 08:41:41.667 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved (None)
home-assistant  | Traceback (most recent call last):
home-assistant  |   File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 658, in state
home-assistant  |     numerical_value = int(value)
home-assistant  |                       ^^^^^^^^^^
home-assistant  | ValueError: invalid literal for int() with base 10: '12H'
home-assistant  |
home-assistant  | During handling of the above exception, another exception occurred:
home-assistant  |
home-assistant  | Traceback (most recent call last):
home-assistant  |   File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 661, in state
home-assistant  |     numerical_value = float(value)
home-assistant  |                       ^^^^^^^^^^^^
home-assistant  | ValueError: could not convert string to float: '12H'
home-assistant  |
home-assistant  | The above exception was the direct cause of the following exception:
home-assistant  |
home-assistant  | Traceback (most recent call last):
home-assistant  |   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 255, in _handle_refresh_interval
home-assistant  |     await self._async_refresh(log_failures=True, scheduled=True)
home-assistant  |   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 411, in _async_refresh
home-assistant  |     self.async_update_listeners()
home-assistant  |   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 165, in async_update_listeners
home-assistant  |     update_callback()
home-assistant  |   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 491, in _handle_coordinator_update
home-assistant  |     self.async_write_ha_state()
home-assistant  |   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1000, in async_write_ha_state
home-assistant  |     self._async_write_ha_state()
home-assistant  |   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1126, in _async_write_ha_state
home-assistant  |     state, attr, capabilities, shadowed_attr = self.__async_calculate_state()
home-assistant  |                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
home-assistant  |   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1061, in __async_calculate_state
home-assistant  |     state = self._stringify_state(available)
home-assistant  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
home-assistant  |   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1006, in _stringify_state
home-assistant  |     if (state := self.state) is None:
home-assistant  |                  ^^^^^^^^^^
home-assistant  |   File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 665, in state
home-assistant  |     raise ValueError(
home-assistant  | ValueError: Sensor sensor.2023_ioniq_5_set_temperature has device class 'temperature', state class 'None' unit '°F' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-numeric value: '12H' (<class 'str'>)
```

**To Reproduce**
Install integration and configure.

**Expected behavior**
Exception not to be raised by temperature_set value.

**Comment by malammar** [comment]
Updating to `2.28.0` may have resolved. I'll close this if it doesn't reoccur.

**Comment by malammar** [comment]
Resolved in 2.28.0
